modules/billing-account/logging.tf

```diff
@@ -37,7 +37,7 @@ resource "google_logging_billing_account_sink" "sink" {
   disabled        = each.value.disabled
 
   dynamic "bigquery_options" {
-    for_each = each.value.type == "biquery" && each.value.bq_partitioned_table != false ? [""] : []
```

Comment:
I don't remember why we depended on partitioned tables, but can you explain why you are removing the check?